zk_toolbox/crates/zk_supervisor/src/commands/test/prover.rs

```diff
@@ -6,7 +6,7 @@ use crate::messages::MSG_PROVER_TEST_SUCCESS;
 
 pub fn run(shell: &Shell) -> anyhow::Result<()> {
     let ecosystem = EcosystemConfig::from_file(shell)?;
-    let _dir_guard = shell.push_dir(&ecosystem.link_to_code.join("prover"));
+    let _dir_guard = shell.push_dir(ecosystem.link_to_code.join("prover"));
```

Comment:
is this related to this PR?

Reply:
well, lints were broken in a different PR (probably here https://github.com/matter-labs/zksync-era/commit/30e072bd695615b0095c8bdcfd62b77c6b0ae5e6), and it prevents this PR from merging